test/.gitignore

```diff
@@ -1,2 +1,2 @@
 test_config.dart
-service-account.json
+service-account.json
```

Comment:
No newline at end of file

Reply:
Why is this necessary?

Reply:
Because if you'd left it there that file wouldn't need to be changed in this commit as there are otherwise no changes there.

Moreover, it's standard practice to leave a newline at the end of every textual file, it's why git notifies you if you forget it.

Reply:
TIL